Network security: rebrand and new elastic cloud UX (#1785)
elastic/docs-content: deploy-manage/security/private-link-traffic-filters.md

applies_to:
  deployment:
    ess: ga
products:
  - id: cloud-hosted
---

# Private connectivity

> Private connectivity is a secure way for your {{ecloud}} deployments and projects to communicate with other cloud provider services over your cloud provider's private network. You can create a virtual private connection endpoint (VCPE) using your provider's private link service. You can also optionally filter traffic to your deployments and projects by creating ingress filters for your VCPE in {{ecloud}}.

Reviewer: ```suggestion
Private connectivity is a secure way for your {{ecloud}} deployments and projects to communicate with other cloud provider services over your cloud provider's private network. You can create a virtual private connection (VPC) using your provider's private link service. You can also optionally filter traffic to your deployments and projects by creating ingress filters for your VPC in {{ecloud}}.
```

applies_to:
  deployment:
    ess: ga
products:
  - id: cloud-hosted
---

# Private connectivity

> Private connectivity is a secure way for your {{ecloud}} deployments and projects to communicate with other cloud provider services over your cloud provider's private network. You can create a virtual private connection (VPC) using your provider's private link service, and then manage it in {{ecloud}} using a private connection policy. You can also optionally filter traffic to your deployments by creating ingress filters for your VPC in {{ecloud}}.

Author: in this context it means "other services offered by/hosted on the same cloud provider", not Azure to AWS or something

| Azure | [Azure Private Link](/deploy-manage/security/azure-private-link-traffic-filters.md) |
| GCP | [GCP Private Service Connect](/deploy-manage/security/gcp-private-service-connect-traffic-filters.md) |

> After you set up your private connection, you can [claim ownership of your VCPE ID](/deploy-manage/security/claim-traffic-filter-link-id-ownership-through-api.md) to prevent other organizations from using it.

Reviewer: ```suggestion
After you set up your private connection, you can [claim ownership of your VCP ID](/deploy-manage/security/claim-traffic-filter-link-id-ownership-through-api.md) to prevent other organizations from using it.
```

To learn how private connection policies work, how they affect your deployment, and how they interact with [IP filter policies](ip-filtering-cloud.md), refer to [](/deploy-manage/security/network-security-policies.md).

:::{tip}
{{ech}} and {{serverless-full}} also support [IP filters](/deploy-manage/security/ip-filtering-cloud.md). You can apply both IP filters and private connections to a single {{ecloud}} resource.
:::

:::{note}

> Private connection policies were formerly referred to as PrivateLink filters.

Reviewer: ```suggestion
Private connection policies were formerly referred to as PrivateLink traffic filters.
```
